modules/cockroachdb/data/cluster_defaults.sql

```diff
@@ -0,0 +1,8 @@
+SET CLUSTER SETTING kv.range_merge.queue_interval = '50ms';
```

Comment:
Looks like there is one change since this advice was introduced in v21.2.17 (a setting was removed in v21.2.22), so looks like we're good. If this was needed we could look to introduce version conditionals.

Reply:
Indeed, although I'd not like to be in the place of maintaining multiple setups, but instead letting users do it in their code. I'm worried about coupling a certain configuration with a certain range of versions that is not documented anywhere. This comment applies to all modules, not only this one.

In any case, we could validate that the image is above a given semver value, also making the docs be very explicit about the `WithNoClusterDefaults` option.